docs/università-vs-self-made/README.md

```diff
@@ -0,0 +1,25 @@
+# Università VS Self-Made
+
+## La terza via: l’ITS
+
+Nel momento in cui si vuole intraprendere la via della “specializzazione” da developer, una volta concluse le scuole superiori, spesso ci si trova davanti a quello che è un bivio: _Intraprendere la via dell’Università o iniziare un percorso di autoformazione?_
+
+La risposta non è sempre così scontata, vista anche la soggettività del tema. Nel primo caso stiamo parlando di un percorso che dura minimo tre anni (o cinque, a seconda dell’obiettivo), strutturato e tenuto da docenti, mentre nel secondo di uno la cui durata è ignota e che si fonda esclusivamente sulla nostra organizzazione, disciplina e motivazione.
+
+In molti però non sono a conoscenza di una terza via, ovvero degli ITS (_Istituto Tecnico Superiore_) di programmazione, degli enti solitamente finanziati dalla regione e/o da alcune aziende del settore, il cui scopo è quello di formare nel giro di un paio d’anni delle figure professionali da inserire nel mondo dell’IT.
+
+### Un ibrido tra Teoria e Pratica
+
+“_Perché l’ITS potrebbe risultare un’ottima via?”_
+
+Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente per la programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.
```

Comment:
Varrebbe la pena specificare meglio su cosa si focalizzano gli ITS (i programmi in generale) e come trovarli

Reply:
Sono d'accordo, effettivamente è dato troppo per scontato. Grazie per avermelo fatto notare. Credo che possa essere una buona idea inserire questa specifica subito dopo il blocco che mi hai evidenziato, visto che riguarda sempre una descrizione del percorso. Cosa ne pensi?

Reply:
@ChristianF11 secondo me hai fatto un buon lavoro integrando il funzionamento degli ITS